openmetadata-ui/src/main/resources/ui/src/pages/AlertsPage/AlertsPage.tsx

```diff
@@ -185,37 +185,34 @@ const AlertsPage = () => {
           </div>
         </Col>
         <Col span={24}>
-          {loading ? (
-            <Loader />
-          ) : (
-            <Table
-              bordered
-              columns={columns}
-              dataSource={alerts}
-              locale={{
-                emptyText: !loading && (
-                  <ErrorPlaceHolder
-                    permission
-                    className="p-y-md"
-                    doc={ALERTS_DOCS}
-                    heading={t('label.alert')}
-                    type={ERROR_PLACEHOLDER_TYPE.CREATE}
-                    onClick={() =>
-                      history.push(
-                        getSettingPath(
-                          GlobalSettingsMenuCategory.NOTIFICATIONS,
-                          GlobalSettingOptions.ADD_ALERTS
-                        )
+          <Table
+            bordered
+            columns={columns}
+            dataSource={alerts}
+            loading={loading}
+            locale={{
+              emptyText: !loading && (
+                <ErrorPlaceHolder
+                  permission
+                  className="p-y-md"
+                  doc={ALERTS_DOCS}
+                  heading={t('label.alert')}
+                  type={ERROR_PLACEHOLDER_TYPE.CREATE}
+                  onClick={() =>
+                    history.push(
+                      getSettingPath(
+                        GlobalSettingsMenuCategory.NOTIFICATIONS,
+                        GlobalSettingOptions.ADD_ALERTS
                       )
-                    }
-                  />
-                ),
-              }}
-              pagination={false}
-              rowKey="id"
-              size="middle"
-            />
-          )}
+                    )
+                  }
+                />
+              ),
+            }}
+            pagination={false}
+            rowKey="id"
+            size="middle"
```

Comment:
We useally use small size any reason we use middle size here?

Reply:
It was already there I haven't changed it, but I will make it small.

Reply:
updated